frontend/src/metabase/admin/tasks/components/Logs/Logs.unit.spec.tsx

```diff
@@ -97,6 +97,20 @@ describe("Logs", () => {
         expect(screen.getByText(errMsg)).toBeInTheDocument();
       });
     });
+
+    it("should stop polling on unmount", async () => {
+      fetchMock.get("path:/api/util/logs", [log]);
+      const { unmount } = render(<Logs />);
+      expect(
+        await screen.findByText(new RegExp(log.process_uuid)),
+      ).toBeInTheDocument();
+
+      unmount();
+      act(() => {
+        jest.advanceTimersByTime(1100); // wait longer than polling period
```

Comment:
**Do not need to change this!**
This is just a general comment - no action needed.

This is fragile.
Once someone changes the default polling period, this test will either break or (even worse) give us false positive.

The correct approach would be to store the polling period in a var and export it from Logs/hooks, and then import it here, and add 100ms to it.

Reply:
That's a great point, some other tests are based on the same polling period of `1000ms`.
It'd be a good idea to refactor this and those other tests too

Reply:
Feel free to open a tech debt issue for it.

Reply:
Here we go #45263 😄